app/src/main/java/com/github/sdp/ratemyepfl/model/review/Review.kt

```diff
@@ -1,78 +1,43 @@
 package com.github.sdp.ratemyepfl.model.review
 
-import com.github.sdp.ratemyepfl.backend.database.RepositoryItem
 import com.github.sdp.ratemyepfl.model.time.DateTime
 import kotlinx.serialization.Serializable
-import kotlinx.serialization.decodeFromString
-import kotlinx.serialization.encodeToString
-import kotlinx.serialization.json.Json
 
 @Serializable
 data class Review constructor(
     val rating: ReviewRating = ReviewRating.AVERAGE,
-    val title: String = "",
-    val comment: String = "",
+    override val title: String = "",
+    override val comment: String = "",
     val reviewableId: String = "",
-    val date: DateTime = DateTime.DEFAULT_DATE_TIME,
-    val uid: String? = null,
-    var likers: List<String> = listOf(),
-    var dislikers: List<String> = listOf()
-) : RepositoryItem {
+    override val date: DateTime = DateTime.DEFAULT_DATE_TIME,
+    override val uid: String? = null,
+    override var likers: List<String> = listOf(),
+    override var dislikers: List<String> = listOf()
+) : Post(title, comment, date, uid, likers, dislikers) {
```

Comment:
Maybe we can specify the title in Review only and not Post. If you have any suggestion, I'm listening :)